chaos-days/blog/2024-07-25-Using-flow-control-to-handle-bottlenecked-exporting/index.md

```diff
@@ -2,66 +2,60 @@
 layout: posts
 title:  "Using flow control to handle bottleneck on exporting"
 date:   2024-07-25
-categories: 
-  - performance 
+categories:
+  - performance
 tags:
   - availability
 authors: rodrigo
 ---
 
-# Using flow control to handle bottleneck on exporting 
+# Using flow control to handle bottleneck on exporting
 
-With the Zeebe now having a unified flow control that protects from user commands that cannot be processed fast enough and from excessive writes, we wanted to test situations when exporting is degraded and how to configure the rate limits for writes to bring the cluster back to a balanced/safe state.
+Zeebe 8.6 introduces a new unified flow control mechanism that is able to limit user commands (by default it tries to achieve 200ms response times) and rate limit writes of new records in general (disabled by default).
```

Comment:
❓ In the Saas cluster plans we have the write limits enabled, should we specify this?

Reply:
We _could_ but I think it's also fine to leave it out as it's internal and most clusters don't have that (yet!).